tools/releaseBuild/Image/DockerFile

```
@@ -18,7 +18,7 @@ RUN Import-Module PackageManagement; `
     Install-ChocolateyPackage -PackageName nuget.commandline -Executable nuget.exe -Cleanup; `
     Install-Module -Force -Name platyPS; `
     Invoke-WebRequest -Uri https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1 -outfile C:/dotnet-install.ps1; `
-    C:/dotnet-install.ps1 -Channel Release -Version 2.2.3; `
+    C:/dotnet-install.ps1 -Channel Release -Version 2.1.4; `

```

Have you tried using the latest patched  version `2.1.9`?
```suggestion
    C:/dotnet-install.ps1 -Channel Release -Version 2.1.9; `
```

---

sadly, that build fails as well. I'm not too fussed about this, as it's just the build tool which is used in the docker image. I can tear this apart after this release.